packages/shared/forks/ReactFeatureFlags.native-oss.js

```diff
@@ -79,6 +79,7 @@ export const enableAsyncActions = false;
 export const alwaysThrottleRetries = true;
 
 export const useMicrotasksForSchedulingInFabric = false;
+export const passChildrenWhenCloningPersistedNodes = __VARIANT__;
```

Comment:
Open problem: I need this to get the test to run both variants, but it fails the OSS RN build

```
/home/circleci/project/build/react-native/implementations/ReactFabric-prod.js
  1317:45  error  '__VARIANT__' is not defined  no-undef
```

Reply:
I think you can just set this one to false right? It should just be off for the non-Meta builds.

Reply:
https://github.com/facebook/react/pull/27489